include/xtt/asn1_utilities.h

```diff
@@ -28,19 +28,19 @@
 extern "C" {
 #endif
 
-#define XTT_X509_CERTIFICATE_LENGTH 276
+#define XTT_X509_CERTIFICATE_LENGTH 332 //276
 size_t xtt_x509_certificate_length(void);
 
 #define XTT_ASN1_PRIVATE_KEY_LENGTH 48
```

Comment:
I think this also needs to be updated (to 53, I think?).

Reply:
We should also add an assert in this function, like we have in the `xtt_x509_from_ed25519_keypair` function, to check the `XTT_X509_CERTIFICATE_LENGTH`.